crates/subspace-service/src/dsn.rs

```diff
@@ -1,7 +1,7 @@
 pub mod import_blocks;
 mod node_provider_storage;
 
-use crate::dsn::node_provider_storage::NodeProviderStorage;
+pub use crate::dsn::node_provider_storage::NodeProviderStorage;
```

Comment:
I'd much prefer to make `node_provider_storage` public to have some structure rather than exporting everything from the root. We already do that with other data structures.